File: documentation-examples-scala/src/scala/dev/typr/foundations/docs/oracle/PaddedStringTypes.scala
package dev.typr.foundations.docs.oracle
import dev.typr.foundationssc.*
import dev.typr.foundationssc.data.*



@SuppressWarnings(Array("unused"))
object PaddedStringTypes:
  //start
  val padded: OracleType[PaddedString] = OracleTypes.charPadded(10)  // CHAR(10)
  val npadded: OracleType[PaddedString] = OracleTypes.ncharPadded(10) // NCHAR(10)
  //stop

File: documentation-examples-java/src/java/dev/typr/foundations/docs/oracle/PaddedStringTypes.java
package dev.typr.foundations.docs.oracle;

import dev.typr.foundations.OracleType;
import dev.typr.foundations.OracleTypes;
import dev.typr.foundations.data.PaddedString;

@SuppressWarnings("unused")
public class PaddedStringTypes {
  // start
  OracleType<PaddedString> padded = OracleTypes.charPadded(10); // CHAR(10)
  OracleType<PaddedString> npadded = OracleTypes.ncharPadded(10); // NCHAR(10)
  // stop
}
